sdk/core/core-lro/review/core-lro.api.md

```diff
@@ -83,6 +153,21 @@ export interface PollOperationState<TResult> {
 // @public
 export type PollProgressCallback<TState> = (state: TState) => void;
 
+// @public
+export type RawHttpHeaders = {
+    [headerName: string]: string;
+};
+
+// @public
+export interface RawResponse {
+    body?: LROBody;
+    headers: RawHttpHeaders;
+    statusCode: number;
+}
+
+// @public
+export const terminalStates: string[];
```

Comment:
out of curiosity, why do we need to expose this?

Reply:
They're being used by `coreClientLRO` and `coreHttpLRO`. Maybe we can get by with hiding it and duplicating there?

Reply:
I hide it.